### ISSUE TITLE
CPU usage on 100% after random interval

### ISSUE THREAD

**Issue by GamerClassN7** [zabbix/zabbix-docker#1505]
##### SUMMARY
after start of the container CPU Usage is normal how ever in random interval after container boot is back to 100%
![image](https://github.com/user-attachments/assets/dcf4f3e3-e93c-42af-bbfd-39e0539ccaf1)
* Each drop is container restart

##### OS / ENVIRONMENT / Used docker-compose files
UBUNTU 24

##### CONFIGURATION
services:
  zabbix-db:
    container_name: zabbix-db
    image: mariadb:11.1
    restart: always
    volumes:
      - ./zabbix-db/mariadb:/var/lib/mysql:rw
      - ./zabbix-db/backups:/backups
    command: --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci
    environment:
      - MYSQL_USER=${MYSQL_USERNAME}
      - MYSQL_PASSWORD=${MYSQL_PASSWORD}
      - MYSQL_ROOT_PASSWORD=REDACTED
    stop_grace_period: 1m
  zabbix-server-mysql:
    container_name: zabbix-server-mysql
    tty: true
    environment:
      - DB_SERVER_HOST=zabbix-db
      - MYSQL_DATABASE=${MYSQL_DATABASE}
      - MYSQL_USER=${MYSQL_USERNAME}
      - MYSQL_PASSWORD=${MYSQL_PASSWORD}
      - MYSQL_ROOT_PASSWORD=REDACTED
      - ZBX_STARTPOLLERS=2
      - ZBX_LOGSLOWQUERIES=3000
    ports:
      - 10051:10051
    restart: unless-stopped
    image: zabbix/zabbix-server-mysql
    depends_on:
      - zabbix-db
  zabbix-frontend:
    container_name: zabbix-frontend
    image: zabbix/zabbix-web-nginx-mysql
    restart: always
    ports:
      - 9988:8080
    environment:
      - ZBX_SERVER_HOST=zabbix-server-mysql
      - DB_SERVER_HOST=zabbix-db
      - MYSQL_DATABASE=${MYSQL_DATABASE}
      - MYSQL_USER=${MYSQL_USERNAME}
      - MYSQL_PASSWORD=${MYSQL_PASSWORD}
      - PHP_TZ=Europe/Prague
    depends_on:
      - zabbix-db
networks: {}

##### STEPS TO REPRODUCE
just start this compose.yml

##### EXPECTED RESULTS
to not consistently use 100% CPU

##### ACTUAL RESULTS
not consistently use 100% CPU

**Comment by dotneft** [comment]
It could be because of known issue with libcurl 8.9.0: https://support.zabbix.com/browse/ZBX-25054 and https://github.com/curl/curl/issues/14280. Please try to use different variant of OS.

**Comment by GamerClassN7** [comment]
After switched to this tag alpine-7.0.* tag i get this error:
![image](https://github.com/user-attachments/assets/d1b352ce-49f8-4c6e-a602-2c4d07afd173)
and i am unable to fix it not iven when returning to original Tag on ubuntu , any suggestions ?

**Comment by dotneft** [comment]
It looks like you upgraded Zabbix to 7.2 version. What tag did you use for Zabbix server?

**Comment by GamerClassN7** [comment]
originally i don't used any tag at all just  `image: zabbix/zabbix-server-mysql` and i tried detrend os as you suggested `zabbix/zabbix-server-mysql:alpine-7.0.2` i don't have experience with dockized ZBX but  on normal Zabbix without docker is database migration run automatically after start of updated server right ? any idea how to fix it, i would rather not lose my data

**Comment by dotneft** [comment]
I checked zabbix/zabbix-server-mysql:alpine-7.0.2, It is 7.0.2 version. Please share log from Zabbix server container.

**Comment by GamerClassN7** [comment]
Hmm how ? container is in restart loop and i am not able to connect to inner shell

**Comment by dotneft** [comment]
```
docker logs <container name>
```

**Comment by GamerClassN7** [comment]
this is the one segment of the loop
```
Starting Zabbix Server. Zabbix 7.0.3 (revision d93ce02).
Press Ctrl+C to exit.

     1:20240905:070423.988 Starting Zabbix Server. Zabbix 7.0.3 (revision d93ce02).
     1:20240905:070423.988 ****** Enabled features ******
     1:20240905:070423.988 SNMP monitoring:           YES
     1:20240905:070423.988 IPMI monitoring:           YES
     1:20240905:070423.988 Web monitoring:            YES
     1:20240905:070423.988 VMware monitoring:         YES
     1:20240905:070423.988 SMTP authentication:       YES
     1:20240905:070423.988 ODBC:                      YES
     1:20240905:070423.988 SSH support:               YES
     1:20240905:070423.988 IPv6 support:              YES
     1:20240905:070423.988 TLS support:               YES
     1:20240905:070423.988 ******************************
     1:20240905:070423.988 using configuration file: /etc/zabbix/zabbix_server.conf
     1:20240905:070423.997 The server does not match Zabbix database. Current database version (mandatory/optional): 07010013/07010013. Required mandatory version: 07000000.
     1:20240905:070423.997 Zabbix Server stopped. Zabbix 7.0.3 (revision d93ce02).
** Preparing Zabbix server
** Preparing database
** Using MYSQL_USER variable from ENV
** Using MYSQL_PASSWORD variable from ENV
** Using MYSQL_ROOT_PASSWORD variable from ENV
********************
* DB_SERVER_HOST: zabbix-db
* DB_SERVER_PORT: 3306
* DB_SERVER_DBNAME: zabbix
********************
** Creating 'zabbix' user in MySQL database
** Database 'zabbix' already exists. Please be careful with database COLLATE!
** Table 'zabbix.dbversion' already exists.
** Preparing Zabbix server configuration file
** Updating '/etc/zabbix/zabbix_server.conf' parameter "ListenIP": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "ListenPort": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "ListenBacklog": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "SourceIP": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "LogType": 'console'...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "LogFile": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "LogFileSize": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "PidFile": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "DebugLevel": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "DBHost": 'zabbix-db'...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "DBPort": '3306'...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "DBSocket": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "DBName": 'zabbix'...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "DBSchema": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "Vault": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "VaultTLSCertFile": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "VaultTLSKeyFile": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "VaultPrefix": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "VaultURL": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "VaultDBPath": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "DBUser": 'zabbix'...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "DBPassword": '****'. Enable DEBUG_MODE to view value ...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "AllowUnsupportedDBVersions": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "MaxConcurrentChecksPerPoller": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "EnableGlobalScripts": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartReportWriters": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "WebServiceURL": 'http://zabbix-web-service:10053/report'...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "HistoryStorageURL": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "HistoryStorageTypes": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "HistoryStorageDateIndex": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StatsAllowedIP": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartPollers": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartIPMIPollers": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartPollersUnreachable": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartTrappers": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartPingers": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartDiscoverers": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartHistoryPollers": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartHTTPAgentPollers": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartHTTPPollers": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartODBCPollers": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartSNMPPollers": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartConnectors": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartPreprocessors": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartTimers": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartEscalators": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartAgentPollers": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartAlerters": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartTimers": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartEscalators": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartLLDProcessors": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "JavaGateway": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "JavaGatewayPort": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartJavaPollers": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartVMwareCollectors": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "VMwareFrequency": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "VMwarePerfFrequency": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "VMwareCacheSize": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "VMwareTimeout": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "SNMPTrapperFile": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartSNMPTrapper": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "SocketDir": '/tmp/'...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "HousekeepingFrequency": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "MaxHousekeeperDelete": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "ProblemHousekeepingFrequency": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "CacheSize": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "CacheUpdateFrequency": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartDBSyncers": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "HistoryCacheSize": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "HistoryIndexCacheSize": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "TrendCacheSize": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "TrendFunctionCacheSize": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "ValueCacheSize": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "Timeout": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "TrapperTimeout": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "UnreachablePeriod": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "UnavailableDelay": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "UnreachableDelay": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "AlertScriptsPath": '/usr/lib/zabbix/alertscripts'...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "ExternalScripts": '/usr/lib/zabbix/externalscripts'...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "FpingLocation": '/usr/sbin/fping'...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "Fping6Location": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "SSHKeyLocation": '/var/lib/zabbix/ssh_keys'...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "LogSlowQueries": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartProxyPollers": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "ProxyConfigFrequency": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "ProxyDataFrequency": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "SSLCertLocation": '/var/lib/zabbix/ssl/certs/'...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "SSLKeyLocation": '/var/lib/zabbix/ssl/keys/'...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "SSLCALocation": '/var/lib/zabbix/ssl/ssl_ca/'...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "LoadModulePath": '/var/lib/zabbix/modules/'...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "TLSCAFile": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "TLSCRLFile": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "TLSCertFile": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "TLSCipherAll": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "TLSCipherAll13": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "TLSCipherCert": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "TLSCipherCert13": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "TLSCipherPSK": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "TLSCipherPSK13": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "TLSKeyFile": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "TLSPSKIdentity": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "TLSPSKFile": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "ServiceManagerSyncFrequency": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "AllowSoftwareUpdateCheck": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "SMSDevices": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "HANodeName": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "NodeAddress": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "User": 'zabbix'...updated
** Updating '/etc/zabbix/zabbix_server.conf' parameter "WebDriverURL": ''...removed
** Updating '/etc/zabbix/zabbix_server.conf' parameter "StartBrowserPollers": ''...removed
Starting Zabbix Server. Zabbix 7.0.3 (revision d93ce02).
Press Ctrl+C to exit.

     1:20240905:070431.385 Starting Zabbix Server. Zabbix 7.0.3 (revision d93ce02).
     1:20240905:070431.385 ****** Enabled features ******
     1:20240905:070431.385 SNMP monitoring:           YES
     1:20240905:070431.385 IPMI monitoring:           YES
     1:20240905:070431.385 Web monitoring:            YES
     1:20240905:070431.386 VMware monitoring:         YES
     1:20240905:070431.386 SMTP authentication:       YES
     1:20240905:070431.386 ODBC:                      YES
     1:20240905:070431.386 SSH support:               YES
     1:20240905:070431.386 IPv6 support:              YES
     1:20240905:070431.386 TLS support:               YES
     1:20240905:070431.386 ******************************
     1:20240905:070431.386 using configuration file: /etc/zabbix/zabbix_server.conf
     1:20240905:070431.395 The server does not match Zabbix database. Current database version (mandatory/optional): 07010013/07010013. Required mandatory version: 07000000.
     1:20240905:070431.395 Zabbix Server stopped. Zabbix 7.0.3 (revision d93ce02).
```

there is not much informations even with DEBUG_LEVER = 4

**Comment by GamerClassN7** [comment]
Hmmm, for some reason after switching to tag: alpine-trunk i am able to start the server, but i am unable to found what this tag mean precisely

**Comment by GamerClassN7** [comment]
but i would rather use fix version and wait for proper release of 7.2 how ever this tag is not available

**Comment by dotneft** [comment]
Zabbix 7.2 is not released yet, and it looks like you upgraded Zabbix server to trunk (7.2), unfortunately it is impossible to revert back changes, only recovery from backup or continue to use 7.2 (trunk images)

**Comment by GamerClassN7** [comment]
> Zabbix 7.2 is not released yet, and it looks like you upgraded Zabbix server to trunk (7.2), unfortunately it is impossible to revert back changes, only recovery from backup or continue to use 7.2 (trunk images)

yes I understand but can I somehow fix version to  to 7.2 since from my understanding tag alpine-trunk is variable -> the latest version i would like to stuck on version 7.2  and not accidently update to new devel version

**Comment by dotneft** [comment]
You can try to use image sha256 sum instead of tag.
